integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/lakehouse/iceberg/IcebergRESTServiceBaseIT.java

```diff
@@ -115,6 +117,7 @@ private void stopSparkEnv() {
     }
   }
 
+  @FormatMethod
```

Comment:
If a method uses `string.format()`, it must be annotated by `@FormatMethod`, or there will be an error.